packages/flutter_tools/test/general.shard/doctor.dart

```diff
@@ -1,60 +0,0 @@
-// Copyright 2014 The Flutter Authors. All rights reserved.
```

Comment:
Can you just delete the one failing test instead of all of them?

Reply:
none of these are testing what they purport to be. The web tests are invalid (now that web is stable the feature flags are effectively ignored), and you should never be able to see multiple desktop platform validators at the same time (how they're shown is dependent on the platform OS).

Finally, when I tried to write a valid test, I couldn't figure out how to mock out the Platform with the testbed runner, as somehow the real platform was leaking through.

Reply:
Also, it was more than 1 failing: https://ci.chromium.org/b/8822482027585120017